engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/registrar/AsyncInterpretationFunction.scala

```diff
@@ -35,6 +36,9 @@ private[registrar] class AsyncInterpretationFunction(val compiledProcessWithDeps
       getRuntimeContext.getExecutionConfig.getParallelism)
   }
 
+
+  override def nodesUsed: List[NodeData] = SplittedNodesCollector.collectNodes(node).map(_.data)
```

Comment:
I know that it is not a part of this change, but can you add a comment in `SplittedNodesCollector.collectNodes` that this logic of collecting should be the same as in `ProcessSplitter`? Also maybe we should move this collecting logic to splitter to make sure that those things will stay consistent?

Reply:
I added comment for now